netci.groovy

```diff
@@ -66,27 +70,31 @@ class WcfUtilities
     
     // Outerloop jobs for WCF Core require an external server reference
     // This should be run 
-    def addWcfOuterloopTestServiceSync(def job, String os, boolean isPR) { 
+    def addWcfOuterloopTestServiceSync(def job, String os, String branch, boolean isPR) { 
         wcfRepoSyncServiceCount++
 
         def operation = isPR ? "pr" : "branch"
 
         job.with { 
             parameters {
+                stringParam('WcfServiceUrl', "http://wcfcoresrv2.cloudapp.net/WcfService${wcfRepoSyncServiceCount}", 'Wcf OuterLoop Test Service Uri')
                 stringParam('WcfRepoSyncServiceUrl', "http://wcfcoresrv2.cloudapp.net/PRService${wcfRepoSyncServiceCount}/pr.ashx", 'Wcf OuterLoop Test PR Service Uri')
             }
         }
-        if (os.toLowerCase().contains("windows")) {
-            job.with { 
-                steps {
-                    batchFile(".\\src\\System.Private.ServiceModel\\tools\\scripts\\sync-pr.cmd ${operation} %WcfRepoSyncServiceUrl%")
-                }           
-            }
-        } 
-        else {
-            job.with { 
-                steps {
-                   shell("HOME=\$WORKSPACE/tempHome ./src/System.Private.ServiceModel/tools/scripts/sync-pr.sh ${operation} \$WcfRepoSyncServiceUrl")
+        // Exclude rc2 branch, since that branch will not have the sync scripts in
```

Comment:
Observation only: this highlights an area we need to consider, namely what can CI assume it can invoke on the PR machine, and where do they live?  I think we now have adequate proof we have an issue to solve (ex: PR's before #xxx could blow up the PR machine, RC2 needs special treatment, etc.) We don't need to solve it now, but I think PR service may need to rely on some kind of bootstrap script executing outside the enlistment that is "always there" and which can delegate into scripts inside the enlistment if they exist.  For later discussion...